internal/http/rdcservice.go

```diff
@@ -393,7 +394,7 @@ func (c *RDCService) DownloadArtifact(jobID, suiteName string, realDevice bool)
 			log.Err(err).Msg("Unable to download artifacts")
 			return artifacts
 		}
-		artifacts = append(artifacts, targetFile)
+		artifacts = append(artifacts, path.Join(targetDir, targetFile))
```

Comment:
Have you tested this? After all the returned `targetFile` is an [absolute path to the destination](https://github.com/saucelabs/saucectl/blob/8df4b5ddaba2d01d93b9cf287bd199b7752b76ce/internal/http/rdcservice.go#L407-L407).

Reply:
Oh I see why I thought it's correct. The default config in `saucectl-espresso-example` is running on VDC with Android emulator. Will revert it.